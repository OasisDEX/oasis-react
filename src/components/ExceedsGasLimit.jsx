import React, { PureComponent } from 'react';
import { PropTypes } from 'prop-types';
// import ImmutablePropTypes from 'react-immutable-proptypes';

const propTypes = PropTypes && {
};
const defaultProps = {};


class ExceedsGasLimit extends PureComponent {
  render() {
    return (
      <div>
        <div>
          Your order exceed gas limit of <b>4.3 Million</b>.
        </div>
        <div>
          To get a working order use the <b>Buy Max Button</b>
        </div>
      </div>
    );
  }
}

ExceedsGasLimit.displayName = 'ExceedsGasLimit';
ExceedsGasLimit.propTypes = propTypes;
ExceedsGasLimit.defaultProps = defaultProps;
export default ExceedsGasLimit;
