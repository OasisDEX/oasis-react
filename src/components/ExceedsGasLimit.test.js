/* global shallow describe it expect */
/* eslint-disable import/first */
import React from 'react';
import { shallow } from 'enzyme';
import ExceedsGasLimit from './ExceedsGasLimit';


describe('(Component) ExceedsGasLimit', () => {
  it('should render', () => {
    const props = {
      children: (
        <span>test</span>
      )
    };
    const wrapper = shallow(
      <ExceedsGasLimit {...props}/>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
