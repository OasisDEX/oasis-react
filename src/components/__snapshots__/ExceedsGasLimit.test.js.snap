// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) ExceedsGasLimit should render 1`] = `
<div>
  <div>
    Your order exceed gas limit of 
    <b>
      4.3 Million
    </b>
    .
  </div>
  <div>
    To get a working order use the 
    <b>
      Buy Max Button
    </b>
  </div>
</div>
`;
